tests/testthat/test-filter-nm-data.R

```diff
@@ -17,12 +17,12 @@ test_that("translate_nm_expr() translates NONMEM filter expressions", {
   test_exprs <- c("SEX==1", "ID.EQ.2", "WT/=70", "AGE.NE.30", "A=1", "WT.GT.40", "B.LE.20")
 
   expect_equal(
-    translate_nm_expr(test_exprs, type = 'accept'),
+    translate_nm_expr(test_exprs, type = 'accept') %>% names(),
```

Comment:
Use `expect_equivalent()` instead avoid checking names.

Reply:
OIC ... that includes the `is.na()` bit.  IMO, we should check the value rather than the names.

Reply:
Yup. I test the values [here](https://github.com/metrumresearchgroup/bbr/blob/3be0ccc1968faf2f6c185fce05765e6f6026a7ad/tests/testthat/test-filter-nm-data.R#L99-L108), but figured removing the NA aspect (since they all have it) would look cleaner in that section, and also more convenient to add additional cases in the future if need be. Let me know what you think about that reasoning.